Library/Formula/postgrest.rb

```diff
@@ -0,0 +1,24 @@
+require "language/haskell"
+
+class Postgrest < Formula
+  include Language::Haskell::Cabal
+
+  desc "Serves a fully RESTful API from any existing PostgreSQL database."
+  homepage "https://github.com/begriffs/postgrest"
+  url "https://github.com/begriffs/postgrest/archive/v0.2.12.0.tar.gz"
+  sha256 "9f7277720b947b06eb53ac0a54686eb437253d417695bc756220e703532a725a"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "postgresql"
+
+  setup_ghc_compilers
+
+  def install
+    install_cabal_package
+  end
+
+  test do
+    system "#{bin}/postgrest", "--help"
```

Comment:
Creating a new database in the temporary directory with `initdb` shouldn't require a password, right?